index.d.ts

```diff
@@ -241,13 +236,14 @@ export interface PaymentCardTextFieldCommonProps {
   cvcPlaceholder?: string;
   disabled?: boolean;
 
-  onChange?: (params: PaymentCardTextFieldNativeEvent) => void;
+  onChange?(
+    event: ReactNative.NativeSyntheticEvent<PaymentCardTextFieldNativeEvent>
+  ): void;
 
-  /** @deprecated */
```

Comment:
I could not find anywhere that documents this as deprecated. It is actually what is used in their example in their docs so I think it is safe to remove the deprecated flag.